### ISSUE TITLE
Behavior of cp changed since 4.14.0

### ISSUE THREAD

**Issue by ranlu** [seung-lab/cloud-files#95]
With cloud-files 4.13.0, if I run `cloudfiles cp test_file /tmp/nonexist/`, cf will create `/tmp/nonexist/` and then copy the file, since 4.14.0 cf fails with a `FileNotFoundError` if the target folder does not exist already. With the newer version, I have to treat local operation differently, while the old behavior was more consistent with other protocols like gs and s3.

**Comment by william-silversmith** [comment]
Hi Ran! 

On MacOS I created the following test case:

```
(ig) wms:~/code/igneous/projects/fly_synapses$ cloudfiles cp hello.txt ./amazing/
(ig) wms:~/code/igneous/projects/fly_synapses$ cloudfiles cp hello.txt /tmp/notexists/
(ig) wms:~/code/igneous/projects/fly_synapses$ ls /tmp/notexists 
hello.txt
```

Have you tried updating to the latest version?

**Comment by ranlu** [comment]
You are right, I pinned the version to avoid issue #92. It seems that issue is also fixed after 4.18. I will close both issues.
